Беспутин Иван

PRIZRAKeee opened the pull request:
Тест первой задачи
[Посмотреть решение](https://hruview.herokuapp.com/PRIZRAKeee/markup-task-2/master/index.html)

PRIZRAKeee committed:
Вариант решения 001

Тест первой задачи

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

PRIZRAKeee committed:
Вариант решения 002

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

PRIZRAKeee committed:
Вариант решения 3

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented:
Замечания по задачам:
9. Не работает озеленение по клику, если ссылка посещенная.
14. Не хватает отступов.
17
![image](https://user-images.githubusercontent.com/11780431/47139861-478c2000-d2d6-11e8-9f4f-38e31098dde9.png)
20 Сделано не для общего случая
![image](https://user-images.githubusercontent.com/11780431/47140009-a356a900-d2d6-11e8-854e-6096720cbaa7.png)

dimastark commented on tasks/11.html:
```diff
@@ -12,7 +12,20 @@ <h1 class="global-heading">&nbsp;</h1>
         <main class="global-results">
             <style>
-                /* your styles here */
+                input[type='email']
```
Думаю можно просто `input`, тут ведь других типов нет.

dimastark commented on tasks/12.html:
```diff
@@ -12,7 +12,16 @@ <h1 class="global-heading">&nbsp;</h1>
         <main class="global-results">
             <style>
-                /* your styles here */
+                .global-scope div:nth-child(2n+1)
```
Думаю можно не писать `div`.

dimastark commented on tasks/13.html:
```diff
@@ -12,7 +12,12 @@ <h1 class="global-heading">&nbsp;</h1>
         <main class="global-results">
             <style>
-                /* your styles here */
+                .global-scope div:nth-child(1) div,
```
Тут можно не писать первый `div`.

dimastark commented on tasks/16.html:
```diff
@@ -12,7 +12,25 @@ <h1 class="global-heading">&nbsp;</h1>
         <main class="global-results">
             <style>
-                /* your styles here */
+                .global-scope div:first-child
```
Думаю можно не писать имя тега.

dimastark commented on tasks/17.html:
```diff
@@ -12,7 +12,10 @@ <h1 class="global-heading">&nbsp;</h1>
         <main class="global-results">
             <style>
-                /* your styles here */
+                .global-scope input[type='checkbox']:checked ~ input[type='checkbox']
```
А просто `input` не подойдет?

dimastark commented on tasks/20.html:
```diff
@@ -12,7 +12,11 @@ <h1 class="global-heading">&nbsp;</h1>
+                .red-gt2:nth-child(3) div,
+                .red-gt2:nth-child(4) div
+                {
+                    background: #ff6698 !important;
```
Давай по новой)

dimastark commented on tasks/5.html:
```diff
@@ -12,7 +12,11 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
```
Давай лучше сделаем классом?

dimastark commented on tasks/8.html:
```diff
@@ -12,7 +12,12 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
-                /* your styles here */
```
Уууу, читерство) Давай сделаем для общего случая.

dimastark reviewed (commented):
🍅 🍅

dimastark commented:
@PRIZRAKeee Будешь доделывать?

PRIZRAKeee committed:
Исправления ошибок

Исправил всё, кроме замечаний по 12, 13 и 16 заданию. Не могу никак придумать, как не использовать тег div

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee committed:
Задание №12

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented:
20 все еще не для общего случая. Будешь доделывать или закрываем без золотой звезды?

PRIZRAKeee commented:
Сейчас переделаю

PRIZRAKeee committed:
Финал

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

PRIZRAKeee committed:
Final 2.0

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

PRIZRAKeee commented:
Всё сделал

dimastark commented:
🍏
